apps/schema/src/components/ColumnEditModal.vue

```diff
@@ -1,180 +1,203 @@
 <template>
   <LayoutModal
-    v-if="show"
-    title="Edit column metadata"
-    @close="cancel"
-    :isCloseButtonShown="!isDisabled"
+      v-if="show === true"
+      title="Edit column metadata"
+      :isCloseButtonShown="false"
```

Comment:
lot's of incorrect lay-outing. next time someone else saves this file it will all be reverted back :(